File: src/main/java/ru/romanow/orders/model/OrderRequest.java
package ru.romanow.orders.model;

import lombok.Data;
import lombok.experimental.Accessors;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.util.List;
import java.util.UUID;

@Data
@Accessors(chain = true)
public class OrderRequest {

    @NotEmpty(message = "{field.is.empty")
    private List<UUID> itemUids;

    @NotEmpty(message = "{field.is.empty")
    private String firstName;

    @NotEmpty(message = "{field.is.empty")
    private String lastName;

    @NotEmpty(message = "{field.is.empty")
    private String address;
}

File: src/main/java/ru/romanow/orders/model/OrderRequest.kt
package ru.romanow.orders.model

import java.util.*
import javax.validation.constraints.NotEmpty

data class OrderRequest(
    @field:NotEmpty(message = "{field.is.empty")
    val itemUids: List<UUID>,

    @field:NotEmpty(message = "{field.is.empty")
    val firstName: String? = null,

    @field:NotEmpty(message = "{field.is.empty")
    val lastName: String? = null,

    @field:NotEmpty(message = "{field.is.empty")
    val address: String? = null
)